modules/bclconvert/.gitignore

```diff
@@ -0,0 +1,2 @@
+bcl-convert
+*.rpm
```

Comment:
do we really need to keep this `.gitignore` here?

Reply:
Why not? We did it with Cellranger. Just preventing foot guns for our future selves.

Reply:
I can take it or leave it. I modeled this after Cellranger. Perhaps keep it around, just to be safe.